health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -766,175 +766,176 @@ export const UICustomizations = {
+                   return;
+                 }
+             const campaignName = row?.name || "";
+             Digit.Utils.campaign.downloadExcelWithCustomName({
+               fileStoreId: fileId,
+               customName: campaignName
+             });
+           };
+ 
+           const onActionSelect = (e) => {
+             if (e.name === "MP_ACTIONS_EDIT_SETUP") {
+               const key = parseInt(row?.additionalDetails?.key);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)